exec/java-exec/src/main/java/org/apache/drill/exec/rpc/user/UserClient.java

```diff
@@ -201,10 +223,12 @@ protected SaslException mapException(Exception e) {
             if (e instanceof ExecutionException) {
               final Throwable cause = Throwables.getRootCause(e);
               if (cause instanceof SaslException) {
-                return new SaslException("Authentication failed: " + cause.getMessage(), cause);
+                return new SaslException(String.format("Authentication failed with encryption context %s with error %s",
```

Comment:
For an end user, this is a dense error message:

> Authentication failed with encryption context Encryption: .. , MaxWrappedChunkSize: .. , MaxRawWrapSendSize ..  with error ..

Could we improve the error message, here and below?